### ISSUE TITLE
ANSI Escape Character Issue

### ISSUE THREAD

**Issue by sphbecker** [nschloe/termtables#1]
Column widths are incorrectly evaluated when escape characters and ANSI color codes are included in the data. The colors are correctly displayed, but the table width is not correct.

**Comment by nschloe** [comment]
Do you have a small example for that?

**Comment by malt3** [comment]
A simple example:
```python3
print(tt.to_string([
    ['ABCD','ABCDEFHIJ'],
    ['key','\033[31mred\033[0m']
]))
```

Returns:
```
┌──────┬──────────────┐
│ ABCD │ ABCDEFHIJ    │
├──────┼──────────────┤
│ key  │ red │
└──────┴──────────────┘
```
![bash_escape](https://user-images.githubusercontent.com/1780588/67731374-3b15c980-f9f8-11e9-8301-c467baa4a353.png)

My suggestion would be to create a version of the cell value with all ANSI escape characters removed and to calculate the string width based on that. Maybe a flag could be added to toggle the functionality.
I can create a pull request with that functionality if implementing this is in your interest as I would like to use the library with colored data.

**Comment by nschloe** [comment]
Fixes in 0.1.1. Thanks for the MWE!
